### ISSUE TITLE
How to create Algolia search index?

### ISSUE THREAD

**Issue by AloisSeckar** [vuejs-translations/docs-cs#28]
Currently the Algoila widget searches EN index

```
// .vitepress\config.ts
algolia: {
      indexName: 'vuejs',
      appId: 'ML0LEBN7FQ',
      apiKey: 'f49cbd92a74532cc55cfbffa5e5a7d01',
     // ...
}
```

How do we setup CS index for czech translation?

**Comment by vitpetricak** [comment]
I think it is necessary to contact Algolia for separate appId/apiKey. https://www.algolia.com/for-open-source/

Or possibly vue-translation can help with providing them?

**Comment by AloisSeckar** [comment]
Thanks for the idea. It turned out the correct contact link is https://docsearch.algolia.com/apply/ (in case someone else is seeking help with this in the future). I filled the form and now we'll see.

**Comment by AloisSeckar** [comment]
Search in czech docs implemented in ffba2cea12c1c8f4461b832c2542d30b3c8ea167 and fb57a2821d81d838c990cae715969569a7d3252c
